docs/node/run-your-node/archive-node.md

```diff
@@ -12,28 +12,112 @@ historic state which is pruned in dump-restore network upgrades.
 
 ## Prerequisites
 
-Archive node requires a pre-existing oasis-node state.
+Running an archive node requires a pre-existing `oasis-node` state.
```

Comment:
What does this mean, if you wanted to spin up an archive node from scratch? Will it sync automatically from other archive nodes or do you need to download the state snapshot from somewhere?

Reply:
No, you need to have state from somewhere.

Reply:
Maybe extend with something like this:

```suggestion
Running an archive node requires a pre-existing `oasis-node` state. If you don't have one,
you can download a snapshot of a specific network state [here][snapshots].

[snapshots]: https://snapshots.oasis.io
```

Reply:
thanks, added